kale/embed/factorization.py

```diff
@@ -310,37 +312,43 @@ def _get_kernel(self, x, y=None):
             raise ValueError("Pre-computed kernel not supported")
         return pairwise_kernels(x, y, metric=self.kernel, filter_params=True, **params)
```

Comment:
To make lives easier, I would actually recommend to just let the argument be a callable rather than directly a function. so in `__init__`

Reply:
or if it is desirable to let it remain like that, maybe allow `__init__` to take all of the supported kernel arguments like `coef0` or `gamma`